packages/devtools_app/lib/src/eval_on_dart_library.dart

```diff
@@ -156,25 +164,176 @@ class EvalOnDartLibrary {
     }
   }
 
-  Future<Library> getLibrary(LibraryRef instance, ObjectGroup isAlive) {
+  Future<Library> getLibrary(LibraryRef instance, Disposable isAlive) {
     return getObjHelper(instance, isAlive);
   }
 
-  Future<Class> getClass(ClassRef instance, ObjectGroup isAlive) {
+  Future<Class> getClass(ClassRef instance, Disposable isAlive) {
     return getObjHelper(instance, isAlive);
   }
 
-  Future<Func> getFunc(FuncRef instance, ObjectGroup isAlive) {
+  Future<Func> getFunc(FuncRef instance, Disposable isAlive) {
     return getObjHelper(instance, isAlive);
   }
 
   Future<Instance> getInstance(
     FutureOr<InstanceRef> instanceRefFuture,
-    ObjectGroup isAlive,
+    Disposable isAlive,
   ) async {
     return await getObjHelper(await instanceRefFuture, isAlive);
   }
 
+  Future<String> getInstanceHashCode(
+    InstanceRef instance, {
+    @required Disposable isAlive,
+  }) async {
+    // TODO(rrousselGit) figure out why sometimes this returns null
+    final hash = await evalInstance(
```

Comment:
If you had the identity hash would that be enough?
@bkonyi would be really handy if all InstanceRef objects had the identity hash code.

Reply:
[Still waiting for a review on that change](https://dart-review.googlesource.com/c/sdk/+/188600).

Reply:
I am rendering what Flutter shows when using Diagnosticable (hash in hex)